src/pages/panel/components/Ambassadors.tsx

```diff
@@ -21,11 +25,23 @@ export default function Ambassadors() {
   const [ambassadorCard, setAmbassadorCard] = useState<string>();
   useChatCommand(
     useCallback(
-      (command: string) => {
+      (command: string, isPrivileged?: boolean) => {
+        if (isPrivileged && command === "refresh") {
+          refreshAmbassadors().then(() => {
+            setAmbassadorCard(undefined);
+            const ambassadorLists = document.querySelectorAll(".scrollbar");
+            ambassadorLists.forEach((list) => {
+              list.scrollTo({
+                top: 0,
+                behavior: "smooth",
+              });
+            });
```

Comment:
Ditto here, refreshing should be in the background and not interrupt the user (unless we happen to remove the ambassador they had selected, so we might need a new useEffect to watch when ambassadors changes and reset the selected card if it no longer exists)